Source/lib/maxicode/decoder/DecodedBitStreamParser.cs

```diff
@@ -95,7 +97,7 @@ internal static DecoderResult decode(byte[] bytes, int mode)
                     break;
             }
 
-            return new DecoderResult(bytes, result.ToString(), null, mode.ToString());
+            return new DecoderResult(bytes, Encoding.UTF8.GetBytes(result.ToString()), result.ToString(), null, mode.ToString());

```

Comment:
In my opinion, that default behavior (UTF8.GetBytes(...)) should better be placed in the Result class because it would be cleaner to see what class really gives the byte data (like the qr code implementation).